src/Fusion.Resources.Functions.Common/ApiClients/PeopleApiClient.cs

```diff
@@ -1,9 +1,6 @@
-﻿using System;
-using System.Net.Http;
-using System.Threading.Tasks;
-using Fusion.Resources.Functions.Integration;
+﻿using Fusion.Resources.Functions.Common.Integration.Http;
 
-namespace Fusion.Resources.Functions.ApiClients;
+namespace Fusion.Resources.Functions.Common.ApiClients;
 
 public class PeopleApiClient : IPeopleApiClient
```

Comment:
I don't think we need PeopleClient at all for this function. If this is something found from the existing function in the resource repo, the use of PeopleApiClient is because it is used in another function

Reply:
There seems to be one reference to the IPeopleApiClient here: Fusion.Resources.Functions.InternalRequestsFunctions:106

Reply:
The idea behind the `Fusion.Resources.Functions.Common` project is to be able to reuse what was originally "resources function" code for the new "summary function".

Reply:
Yes, that makes sense. In this case we don't need the things related to the other functions. Only the stuff related to Summary notifications should be "merged" over to the new functions.

Reply:
Sorry - This was a misread from my part. Looks good!